From 414534c8260b3bd329cb653aef6640037dd7c4ea Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Tue, 19 Nov 2024 20:43:33 +0100
Subject: [PATCH] [DogeCash] Fix `chronik_block_header.py` (#50)

This should use the Dogecoin blockheader.

(cherry picked from commit 80c638c5bc3b9d91e682348d5916720d3969650c)

diff --git a/test/functional/chronik_block_header.py b/test/functional/chronik_block_header.py
index 48615ab031581d1f5be87579790da7299b37ca51..f0689059029ff5f56a801345d19c52b8daa77a7a 100644
--- a/test/functional/chronik_block_header.py
+++ b/test/functional/chronik_block_header.py
@@ -15,8 +15,8 @@ from test_framework.util import assert_equal, hex_to_be_bytes
 NUM_HEADERS = 10
 
 GENESIS_HEADER = bytes.fromhex(
-    "0100000000000000000000000000000000000000000000000000000000000000000000003ba3edfd7"
-    "a7b12b27ac72c3e67768f617fc81bc3888a51323a9fb8aa4b1e5e4adae5494dffff7f2002000000"
+    "010000000000000000000000000000000000000000000000000000000000000000000000696ad20e2d"
+    "d4365c7459b4a4a5af743d5e92c6da3229e6532cd605f6533f2a5bdae5494dffff7f2002000000"
 )
 
 
